doc/PodioInputOutput.md

```diff
@@ -18,115 +18,218 @@ limitations under the License.
 -->
 # Reading and writing EDM4hep files in Gaudi
 
-The facilities to read and write EDM4hep (or in general event data models based
-on podio) are provided by [`k4FWCore`](https://github.com/key4hep/k4FWCore).
-This page will describe their usage, but not go into too much details of their
-internals. This page also assumes a certain familiarity with Gaudi, i.e. most of
-the snippets just show a minimal configuration part, and not a complete runnable
-example.
+The facilities to read and write EDM4hep (or in general event data models based on podio) are provided by [k4FWCore](https://github.com/key4hep/k4FWCore). This page will describe their usage, but not go into too much details of their internals. This page also assumes a certain familiarity with Gaudi, i.e. most of the snippets just show a minimal configuration part, and not a complete runnable example.
 
-## The `k4DataSvc`
+## Accessing event data
 
-Whenever you want to work with EDM4hep in the Gaudi based framework of Key4hep,
-you will need to use the `k4DataSvc` as *EventDataSvc*. You can instantiate and
-configure this service like the following
+`IOSvc` is an external Gaudi service for reading and writing EDM4hep and other PODIO-based data models. The service should be imported from `k4FWCore` and named "IOSvc" as other components may look for it under this name.
 
 ```python
-from Gaudi.Configuration import *
-from Configurables import k4DataSvc
+from k4FWCore import IOSvc
 
-evtSvc = k4DataSvc("EventDataSvc")
+io_svc = IOSvc("IOSvc")
```

Comment:
I don't know if we need / want to mention this explicitly in the documentation but I think `"IOSvc"` is pretty much required as name for the `IOSvc`, right? At least some of the CLI examples below will only work with that. Can we create an empty `IOSvc()` and rely on a default name?

Reply:
I checked to make sure and I didn't see any errors when using a different name or no name at all for `IOSvc`. Only the `Reader` and `Writer` retrieve it through a `ServiceHandle`.